backend/Dockerfile

```diff
@@ -52,17 +50,20 @@ RUN pip install --no-cache psycopg[binary] gunicorn
 # copy project
 COPY --chown=app:app . $APP_HOME
 
-# change to the app user
-USER app
+# # change to the app user
+# USER app
 
 ARG SECRET_KEY
 ENV SECRET_KEY=${SECRET_KEY}
 ENV DJANGO_SETTINGS_MODULE=backend.production_settings
 
+# RUN python manage.py collectstatic --no-input --clear
+# RUN sleep 3
+
 # run entrypoint.prod.sh
-EXPOSE 8000
+EXPOSE 80
```

Comment:
The port should match the same used in line 66 of this file, which is currently 8000.

Reply:
Nginx is in this pod now so that's the port that the outside is going to use.